how-to/security/openssh-server.md

```diff
@@ -65,35 +67,61 @@ Many other configuration directives for `sshd` are available to change the serve
 
 ## SSH keys
 
-SSH allows authentication between two hosts without the need of a password. SSH key authentication uses a **private key** and a **public key**.
+SSH allows authentication between two hosts without the need of a password, using cryptographic keys instead.
+{term}`SSH-key` authentication uses a **private key** and a **public key**.
+
+We recommend the **ed25519 Elliptic Curve algorithm** (`-t ed25519`) due to shorter keysize and lower computational requirements.
+Alternatively, you can create a key using the **RSA Algorithm** (`-t rsa -b 4096`, for 4096 bit keysize) instead.
 
-To generate the keys, run the following command:
+To generate a key pair, run the following command:
 
 ```bash
-ssh-keygen -t rsa
+ssh-keygen -t ed25519
 ```
 
-This will generate the keys using the **RSA Algorithm**. At the time of this writing, the generated keys will have 3072 bits. You can modify the number of bits by using the `-b` option. For example, to generate keys with 4096 bits, you can use:
+During creation, you will be prompted for a key password, which you would enter to use that key (every time, or instead using an [ssh agent](#ssh-agent).
```

Comment:
s/key password/key passphrase/

Or just s/key password/passphrase/, to avoid repeating "key" twice.